go.mod

```diff
@@ -1,97 +1,80 @@
 module plane.watch
 
-go 1.21
+go 1.23.0
```

Comment:
No 1.24.1? :P

Reply:
heh, fedora only has go version 1.23 atm - so I guess this is why :D